lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -10,9 +10,9 @@ module Common
     DOC_CONFLICT_CODE = 409
 
     # When you use external versioning, you are communicating that you want
-    # to ignore conflicts. More obviously, since an external version is a 
```

Comment:
@guyboertje my editor seems to cleanup unneeded space, if its a problem for merging, I can restore them

Reply:
No need to worry in this case.